src/Tribe/Admin/Settings.php

```diff
@@ -246,6 +247,75 @@ public function add_admin_pages() {
 		$this->maybe_add_app_shop();
 	}
 
+	/**
+	 * Redirects users to the First Time Setup page when accessing any TEC settings or management page for the first time.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function redirect_tec_pages_to_first_time_setup(): void {
+		// If there is more than one previous version, don't redirect since they're probably already setup.
```

Comment:
"setup" is a noun. This probably should say "since they're probably already set up."

Reply:
What Andras said - also I'd use they have instead of they are for this:

```suggestion
		// If there is more than one previous version, don't redirect since they've probably already set up.
```